command/agent/consul/mock.go

```diff
@@ -0,0 +1,27 @@
+package consul
+
+import (
+	"log"
+
+	"github.com/hashicorp/consul/api"
```

Comment:
we have been following the habit of <name>_testing.go to put importable stubs/mocks/etc